script/BaseYnEigenScript.s.sol

```diff
@@ -1,5 +1,6 @@
 // SPDX-License-Identifier: BSD 3-Clause License
```

Comment:
let's move this file to the ynEigen directory

Reply:
fixed in [498985f](https://github.com/yieldnest/yieldnest-protocol/pull/158/commits/498985f751fe9f13299aaba4a6dd055950a650a6)